### PR TITLE
Added dot-portlet-box in content types to frame the iframes

### PR DESCRIPTION
**Pull request by ivorpad**
Fixed the issue reported in https://github.com/dotCMS/core/issues/19824

### CODE REVIEW

File: src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-row/content-type-fields-row.component.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 @use "variables" as *;
 
 :host {
-    border-radius: $border-radius;
```
Comment (ivorpad) [inline src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-row/content-type-fields-row.component.scss:4]: To maintain consistency with `dot-portlet-box`

File: src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html [hunk 2]
```diff
@@ -35,9 +35,9 @@ <h4>{{ contentType.name }}</h4>
         header="{{ 'contenttypes.tab.fields.header' | dm }}"
     >
         <div class="content-type__fields-layout" id="content-type-form-layout">
-            <div class="content-type__fields-main" id="content-type-form-main">
+            <dot-portlet-box class="content-type__fields-main">
```
Comment (ivorpad) [inline src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:38]: As per DMs on Slack, I don't think it's necessary to refactor this component as there might be too many edge cases. I decided to target with a CSS class instead.

File: src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html [hunk 3]
```diff
@@ -35,9 +35,9 @@ <h4>{{ contentType.name }}</h4>
         header="{{ 'contenttypes.tab.fields.header' | dm }}"
     >
         <div class="content-type__fields-layout" id="content-type-form-layout">
```
Comment (fmontes) [inline src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:37]: Can we get rid of this `div` in favor of just `dot-portlet-box`?

File: src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html [hunk 4]
```diff
@@ -35,9 +35,9 @@ <h4>{{ contentType.name }}</h4>
         header="{{ 'contenttypes.tab.fields.header' | dm }}"
```
Comment (fmontes) [inline src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:35]: All the new `dot-portlet-box` needs tests.

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by alfredo-dotcms — DISMISSED** [review]

**Review by hmoreras — APPROVED** [review]

**Pull request merged**